legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabOAuthAuthenticator.java

```diff
@@ -355,9 +355,9 @@ private String getCodeFromLocation(String location, String expectedState)
     {
         int redirectURILength = getAppRedirectURI().length();
         char nextChar = location.charAt(redirectURILength);
-        if (nextChar != '#')
+        if (nextChar != '?')
```

Comment:
Does this means the code wont be backward  compatible?  Do we need to manage both `#` and `?`?

Reply:
no we don't need to, we are completely demising the flow